cristin-person/src/main/java/no/unit/nva/cristin/person/client/AuthorizedCristinPersonApiClient.java

```diff
@@ -0,0 +1,57 @@
+package no.unit.nva.cristin.person.client;
+
+import no.unit.nva.cristin.person.model.cristin.CristinPerson;
+import no.unit.nva.cristin.person.model.nva.Person;
+import no.unit.nva.utils.UriUtils;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.core.paths.UriWrapper;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpResponse;
+import java.util.List;
+
+import static java.util.Arrays.asList;
+import static no.unit.nva.cristin.model.Constants.CRISTIN_API_URL;
+import static no.unit.nva.cristin.model.Constants.PERSON_CONTEXT;
+import static no.unit.nva.cristin.person.client.CristinPersonQuery.CRISTIN_API_PERSONS_PATH;
+import static no.unit.nva.utils.UriUtils.PERSON;
+
+public class AuthorizedCristinPersonApiClient extends CristinPersonApiClient {
+
+    public static final String NATIONAL_ID = "national_id";
```

Comment:
Have this just one place and import it elsewhere. Found it also in CristinPersonQuery